_search-plugins/sql/settings.md

```diff
@@ -79,6 +79,7 @@ Setting | Default | Description
 `plugins.query.memory_limit` | 85% | Configures the heap memory usage limit for the circuit breaker of the query engine.
 `plugins.query.size_limit` | 200 | Sets the default size of index that the query engine fetches from OpenSearch.
 `plugins.query.datasources.enabled` | true | Change to `false` to disable support for data sources in the plugin.
+`plugins.query.field_type_tolerance` | true | set to false, then an array is reduced to the first non array value of any level of nesting.
```

Comment:
Can we add info about what happens if this setting is `true`? Does OpenSearch still accept the array without attempting to reduce it to a single element? And what is the default?

Reply:
add settingt is true description, https://github.com/opensearch-project/documentation-website/pull/8646/files#diff-9d8dd46675baec61a9c50d9e2bb1054470a532c39ae8752e1f4e48b3e49c780eR82
default value is true.

more example in https://github.com/opensearch-project/sql/blob/main/docs/user/admin/settings.rst#plugins-query-field-type-tolerance